File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        store = set(nums)
        res = 0
        
        for i in nums:
            lcs, curr = 0, i
            while i in store:
                lcs += 1
                i += 1
            res = max(res, lcs)
        return res

File: Data Structures & Algorithms/pow-x-n/submission-0.cpp
class Solution {
public:
    double myPow(double x, int n) {
        if(n == 0) return 1;
        if(x == 1) return 0;

        double res = 1;
        for(int i = 0; i < abs(n); i++){
            res *= x;
        }
        return n >= 0 ? res : 1 / res;
    }
};
